spec/models/validations/sales/setup_validations_spec.rb

```diff
@@ -105,6 +105,52 @@
           expect(record.errors[:saledate]).to include("Enter a date within the 23/24 or 24/25 collection years, which is between 1st April 2023 and 31st March 2025")
         end
       end
+
+      context "when after the new logs end date but before edit end date for the previous period" do
```

Comment:
I'm guessing the linter want you to put 'when' before 'after' even though this makes sense starting with after :)

Reply:
Is this when a sales log's sale date is after the new logs end date? I'm not sure how to reword this but I don't think I understand it without looking at the contents.